Automattic/wp-calypso
# Signup: Reduxify the Verticals V2 step

This PR aims to add Redux support to the Verticals V2 step.

It currently only saves the data to the Redux store without using it anywhere.

This will be added at a later point. 

See #8718.

To test:

1. Check out step or use Calypso.live
2. Go through signup and make sure to check the Verticals V2 
3. After submitting the step check if the Verticals V2 step is saved in the Indexed DB under `signup.steps.survey`

cc @michaeldcain @coreh
